prow/external-plugins/cherrypicker/server_test.go

```diff
@@ -1172,6 +1172,16 @@ func TestReleaseNoteFromParentPR(t *testing.T) {
 			input:    "```feature developer #999\nUpdate the magic number from 42 to 49\n```",
 			expected: "```feature developer github.com/foo/bar #123 @foo-author\nUpdate the magic number from 42 to 49\n```",
 		},
+		{
+			name:     "test9",
+			input:    "```feature developer\nUpdate the magic number from 42 to 49\n```\n\n```feature operator\nUpdate another magic number\n```",
```

Comment:
Small q: I see that there are two newlines between the release notes. Is this intentional for the test? Usually we only separate multiple release notes with 1 new line.

Reply:
I removed the second `\n` 😄